release/docker/Dockerfile

```diff
@@ -25,5 +25,9 @@ RUN mv /usr/share/$ARCHIVE_FILE_UNPACKED /usr/share/data-prepper
 COPY default-data-prepper-config.yaml $ENV_CONFIG_FILEPATH
 COPY default-keystore.p12 /usr/share/data-prepper/keystore.p12
 
+# Do not run as root, but as user "nobody" (uid 65534)
+RUN chown -R nobody:nobody $DATA_PREPPER_PATH /var/log/data-prepper
```

Comment:
It seems that best practices are to create a purpose-built user for running Docker containers.

How about doing this instead?

```
RUN addgroup -S data_prepper_group
RUN adduser -S data_prepper -G data_prepper_group
RUN chown -R data_prepper_group:data_prepper_group $DATA_PREPPER_PATH /var/log/data-prepper
USER data_prepper
```

Reply:
Done, I used a simpler variant that creates group automatically, used a no-login shell and no creation of home directory, which is similar to the `nobody` user and sligtly more secure. Specified UID 1000 for easier referral in `runAsUser` etc.